docs/guidance/mobilenet-rayservice.md

```diff
@@ -0,0 +1,42 @@
+# RayService: MobileNet example
+
+Both Python scripts for Ray Serve and sending requests are in the repository [kevin85421/ray-serve-examples](https://github.com/kevin85421/ray-serve-examples).
+
+## Step 1: Create a Kubernetes cluster with Kind.
+
+```sh
+kind create cluster --image=kindest/node:v1.23.0
+```
+
+## Step 2: Install KubeRay operator
+
+Follow this [document](../../helm-chart/kuberay-operator/README.md) to install the latest stable KubeRay operator via Helm repository.
+
+## Step 3: Install a RayService
+
+```sh
+# path: ray-operator/config/samples/
+kubectl apply -f ray-service.mobilenet.yaml
+```
+
+* The [mobilenet.py](https://github.com/kevin85421/ray-serve-examples/blob/main/mobilenet.py) requires `tensorflow` as a dependency. Hence, the YAML file uses `rayproject/ray-ml:2.5.0` instead of `rayproject/ray:2.5.0`.
+* `python-multipart` is required for the request parsing function `starlette.requests.form()`, so the YAML file includes `python-multipart` in the runtime environment.
+
+## Step 4: Forward the port of Serve
+
+```sh
+kubectl port-forward svc/rayservice-mobilenet-serve-svc 8000
+```
+
+Note that the Serve service will be created after the Serve applications are ready and running. This process may take some time.
```

Comment:
Could you give a rough estimate for how long the process should take? Does "some time" mean minutes or hours?

Reply:
I performed an experiment:

1. Load the Ray image into the Kind cluster, so that all Pods in the RayCluster will start running within a few seconds.
2. Create the RayService
3. The Serve service is created after 39 seconds (42s - 3s = 39s) when all Pods are running.

<img width="1312" alt="Screen Shot 2023-06-20 at 10 14 03 PM" src="https://github.com/ray-project/kuberay/assets/20109646/8acaf028-7550-47fd-9418-e188282cf104">

Reply:
https://github.com/ray-project/kuberay/pull/1175/commits/317004637c879781737b62fe9cd8455c2bd1c195